src/riemann.js

```diff
@@ -97,14 +97,16 @@ export default class Riemann {
   handlePacketEvent(eventString) {
     // See here for valid event properties:
     // http://aphyr.github.io/riemann/concepts.html
-    this.send({
-      service: this.getService(eventString),
-      state: 'ok',
-      description: this.getDescription(eventString),
-      tags: this.getTags(eventString),
-      metric: this.getMetric(eventString),
-      ttl: this.config.ttl
-    });
+    if (eventString !== undefined && eventString != '') {
```

Comment:
Sure.  I was being explicit but I'm fine with that.  Will submit a change shortly.